pkg/snmp/traps/formatter.go

```diff
@@ -187,6 +188,23 @@ func NormalizeOID(value string) string {
 	return strings.TrimLeft(value, ".")
 }
 
+// IsValidOID returns true if a looks like a valid OID.
+// An OID is made of digits and dots, but OIDs do not end with a dot and there are always
+// digits between dots.
+func IsValidOID(value string) bool {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think this function should be unit tested alone with all possible type of cases.

Reply:
Just making sure you have seen this test here: https://github.com/DataDog/datadog-agent/pull/12832/files#diff-4fa2128430993df9fe7e767f12556395aae5e8ae352e82eed340358a6e91cf05R303

What would you add to it?

Reply:
There is `TestIsValidOID`, but I think we should test for this function should be in `formatter_test.go` since that's most golang devs will expect the test to be.

Reply:
Ah good point, I moved the function but forgot to move the test 👍